READMEv2.md

```diff
@@ -0,0 +1,212 @@
+![Versatile Data Kit](./support/images/versatile-data-kit.svg#gh-light-mode-only)
+![Versatile Data Kit](./support/images/versatile-data-kit.svg#gh-dark-mode-only)
+
+<p align="center">
+    <a href="https://github.com/vmware/versatile-data-kit/pulse" alt="Activity">
+        <img src="https://img.shields.io/github/commit-activity/m/vmware/versatile-data-kit" /></a>
+    <a href="https://github.com/vmware/versatile-data-kit/contributors" alt="Last Activity">
+        <img src="https://img.shields.io/github/last-commit/vmware/versatile-data-kit" alt="Last Activity"></a>
+    <a href="https://github.com/vmware/versatile-data-kit/blob/main/LICENSE" alt="License">
+        <img src="https://img.shields.io/github/license/vmware/versatile-data-kit" alt="license"></a>
+    <a href="https://github.com/pre-commit/pre-commit">
+        <img src="https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white" alt="pre-commit"></a>
+    <a href="https://github.com/vmware/versatile-data-kit">
+        <img src="https://gitlab.com/vmware-analytics/versatile-data-kit/badges/main/pipeline.svg" alt="build status"></a>
+    <a href="https://twitter.com/intent/tweet?text=Wow: @VDKProject">
+        <img src="https://img.shields.io/twitter/url?style=social&url=https%3A%2F%2Ftwitter.com%2FVDKProject" alt="twitter"/></a>
+     <a href="https://www.youtube.com/channel/UCasf2Q7X8nF7S4VEmcTHJ0Q">
+        <img alt="YouTube Channel Subscribers" src="https://img.shields.io/youtube/channel/subscribers/UCasf2Q7X8nF7S4VEmcTHJ0Q?style=social"></a>
+
+<!-- TODO: code coverage -->
+</p>
+
+
+<p align="center">
+  <span> 🧑‍💻 Develop </span> <span> ▶️ Run </span> <span> 📊 Manage </span> <br>
+   <span>data workloads<span>
+</p>
+
+---
+
+<div align="left">
```

Comment:
It was on purpose - left, right,center, left,right. Trying to make it more interesting looking. But I see I am failing at that since it's mostly confusing.

Reply:
I change it not to look so buggy. I think it's ok now.

Reply:
no its still a bit confusing. I think they should all be left aligned.

Reply:
Looks ok to me. I like that the SDK benefits are ont he left and Control Service benefits are on the right.

Reply:
I tried to describe something like this (excuse the bad photo edit)

![Screenshot 2023-08-03 at 14 52 13](https://github.com/vmware/versatile-data-kit/assets/91800778/8a40aa6a-d5cf-4796-8a16-831a8d62d066)

Not sure if it's doable in markdown.